mountpoint-s3-fs/src/metrics.rs

```diff
@@ -310,4 +360,86 @@ mod tests {
             }
         });
     }
+
+    #[test]
+    #[ignore]
```

Comment:
Does it make sense for us to have tests that are used for manual testing and will be ignored in the code base? Not clear what the standard for this is. If we decide to keep them, maybe we should make the purpose clear with comments

Reply:
Also, should we have a test for the case where the config is none?